package.json

```diff
@@ -31,10 +31,10 @@
     "repl": "node tools/repl.js",
     "release": "ts-node-script scripts/release/cli.ts",
     "pretest": "node tools/pretest.js",
-    "test": "lerna run --concurrency 4 --stream test",
-    "test:ci": "lerna run --concurrency 4 --stream test:ci",
-    "test:release": "lerna run --concurrency 4 --ignore @firebase/*-exp --ignore firebase-exp --stream test:ci",
-    "test:exp": "lerna run --concurrency 4 --stream --scope @firebase/*-exp --scope firebase-exp test",
+    "test": "lerna run --concurrency 4 test",
```

Comment:
Why turn off stream for regular (non-CI) tests? Seems helpful to see progress as it goes?

Reply:
Added it back.